pkg/distributor/distributor_test.go

```diff
@@ -5690,17 +5687,19 @@ func makeWireChunk(c chunk.EncodedChunk) client.Chunk {
 }
 
 func (i *mockIngester) QueryStream(ctx context.Context, req *client.QueryRequest, _ ...grpc.CallOption) (client.Ingester_QueryStreamClient, error) {
+	i.trackCall("QueryStream")
```

Comment:
Note to reviews: moving call to `trackCall()` fixes `TestDistributor_QueryStream_ShouldSupportIngestStorage()` flakyness. Why? Because when ingester requests are NOT minimized, the context gets canceled as soon as we reach quorum. There are cases where it gets canceled before `enforceReadConsistency()` is called. In such cases, the `trackCall()` was not called.